packaging/document.conf

```diff
@@ -23,6 +23,13 @@ logdir = /var/log/foundationdb/document
 # implicit-transaction-timeout = 7000
 # pipeline = aggressive 
 # slow-query-log = all
+# metric_plugin = /path/to/lib
+# metric_plugin_config = /path/to/config/file
+# tls_ca_file = /path/to/CA/file
+# tls_key_file = /path/to/key/file
+# tls_certificate_file = /path/to/cert/file
+# tls_password = password string
```

Comment:
I would remove this `tls_password = password string` though, because that's maybe giving the wrong hint

Reply:
Will do.

Reply:
3798323d78fc3933f16ccc1f54d9e8cbb5ceba1a